src/main/java/uk/gov/hmcts/reform/sscs/config/jms/TribunalsHearingsCustomMessageConverter.java

```diff
@@ -0,0 +1,39 @@
+package uk.gov.hmcts.reform.sscs.config.jms;
+
+import com.fasterxml.jackson.databind.ObjectWriter;
+import org.apache.qpid.jms.message.JmsBytesMessage;
+import org.apache.qpid.jms.provider.amqp.message.AmqpJmsMessageFacade;
+import org.apache.qpid.proton.amqp.Symbol;
+import org.jetbrains.annotations.NotNull;
+import org.springframework.jms.support.converter.MappingJackson2MessageConverter;
+import org.springframework.jms.support.converter.MessageType;
+import org.springframework.stereotype.Component;
+
+import java.io.IOException;
+import javax.jms.BytesMessage;
+import javax.jms.JMSException;
+import javax.jms.Session;
+
+@Component
+public class TribunalsHearingsCustomMessageConverter extends MappingJackson2MessageConverter {
```

Comment:
You can call this something like JsonMessage converter, nothing but the name ties it to Tribunals Hearings

Reply:
Good shout did have a generic name before but renamed it based on it only being used within my queue but have updated now to Generic name